src/utils/constants.ts

```diff
@@ -1,12 +1,13 @@
-export const SPONSOR_LIST_FILTER_TAG_KEY = "FRONT24";
+export const SPONSOR_LIST_FILTER_TAG_KEY = "FRONT22";
```

Comment:
Could we change all these to `FRONNT24` so we can test the proper content?

Reply:
Ah, just saw the other PR. 👍

Reply:
It probably makes sense to merge the other one first and then update this one, right?

Reply:
yeah - done

Reply:
LGTM after rebasing. 👍 (I won't be able to give any feedback later today or tomorrow)